python/utils/LinkedLibraryHolder.cpp

```diff
@@ -253,15 +253,7 @@ LinkedLibraryHolder::getPlatform(const std::string &platformName) {
       std::string("getQuantumPlatform_") + platformName);
 }
 
-void LinkedLibraryHolder::resetTarget() {
-  // TODO: create config for default target and use setTarget("qpp") here,
-  // instead of having this be a code duplication of the logic below.
-  __nvqir__setCircuitSimulator(getSimulator("qpp"));
-  auto *platform = getPlatform("default");
-  platform->setTargetBackend("qpp");
-  setQuantumPlatformInternal(platform);
-  currentTarget = "default";
-}
+void LinkedLibraryHolder::resetTarget() { setTarget("qpp-cpu"); }
```

Comment:
I see the TODO comment, but it looks like `setTarget` is quite a bit different than the original `resetTarget` code. Are we sure this is functionally equivalent? e.g., the `disallowTargetModification` could put a kink in things.

Reply:
Will discuss with Bettina.

Reply:
This update looks fine.